src/pages/home/Home.tsx

```diff
@@ -27,7 +27,7 @@ const Home: FunctionComponent = () => {
   const [isMobileOpen, setMobileOpen] = useState(false);
   return (
     <Box
-      sx={ { display: "flex", flexGrow: 1, background: theme.colors.black100 } }
+      sx={ { backgroundColor: theme.colors.black100, display: "flex", flexGrow: 1, height: "fit-content" } }
```

Comment:
This line fixes the issue of the Home container not expanding the full height when content is more than the fold